app/apphandlers/gatewayAppHandler.go

```diff
@@ -89,6 +89,20 @@ func (a *gatewayApp) StartRudderCore(ctx context.Context, options *app.Options)
 	}
 	defer gatewayDB.Stop()
 
+	errDB := jobsdb.NewForWrite(
+		"proc_error",
+		jobsdb.WithClearDB(options.ClearDB),
+		jobsdb.WithDSLimit(&a.config.gatewayDSLimit), // TODO: use better config
```

Comment:
`dsLimit` is not of much use for a writer jobsdb. This is an option that affects queries

Reply:
Removed.